psl/parser-database/src/walkers.rs

```diff
@@ -60,6 +60,15 @@ impl crate::ParserDatabase {
             .map(|enum_id| self.walk(enum_id))
     }
 
+    /// Find a model by name.
+    pub fn find_model<'db>(&'db self, name: &str) -> Option<ModelWalker<'db>> {
+        self.interner
+            .lookup(name)
+            .and_then(|name_id| self.names.tops.get(&name_id))
+            .and_then(|top_id| top_id.as_model_id())
+            .map(|model_id| self.walk(model_id))
+    }
+
```

Comment:
where is this used?

Reply:
Next PR :p I extracted all the changes in the crate and that came with. I'm fine with deleting it in this PR, but the method is general purpose enough that I don't mind which PR it gets in.

Reply:
It doesn't seem to currently be used, I can only find references for `find_enum` 🤔

Reply:
No need to delete.

Reply:
ahhh, github didn't show me tom's message before I responded 😅